treeherder/model/derived/jobs.py

```diff
@@ -731,30 +723,27 @@ def get_result_set_details(self, result_set_ids):
 
         return aggregate_details
 
-    def set_lower_tier_signatures(self):
+    def _get_lower_tier_signatures(self):
         # get the lower tier data signatures for this project.
         # if there are none, then just return an empty list
         # this keeps track of them order (2, then 3) so that the latest
         # will have precedence.  If a job signature is in both Tier-2 and
         # Tier-3, then it will end up in Tier-3.
-        self.lower_tier_signatures = []
+        lower_tier_signatures = []
         for tier_num in self.LOWER_TIERS:
```

Comment:
Oh, I actually meant to delete the value from ExclusionProfileManager.

To be honest, I think all this exclusion profile stuff needs a rethink (as @edmorley mentioned in the bug). I think I'd rather just keep the define in the jobs model for now as it sort of has a built in semantic of "big blob of stuff that needs to be broken up and refactored". :)